PyViCare/PyViCareRadiatorActuator.py

```diff
@@ -14,12 +13,27 @@ def getBatteryLevel(self) -> int:
 
     @handleNotSupported
     def getTemperature(self):
-        return self.service.getProperty("device.sensors.temperature")["properties"]["value"]["value"]
+        # Zigbee TRV reports temperature under trv.temperature, not device.sensors.temperature
+        return self.service.getProperty("trv.temperature")["properties"]["value"]["value"]
+
+    @handleNotSupported
+    def getValvePosition(self):
+        return self.service.getProperty("trv.valve.position")["properties"]["position"]["value"]
+
+    @handleNotSupported
+    def getChildLockStatus(self):
+        return self.service.getProperty("trv.childLock")["properties"]["status"]["value"]
+
+    @handleNotSupported
+    def getMountingModeActive(self):
+        return self.service.getProperty("trv.mountingMode")["properties"]["active"]["value"]
+
+    @handleNotSupported
+    def getLinkQuality(self):
+        return self.service.getProperty("device.zigbee.lqi")["properties"]["strength"]["value"]
```

Comment:
Maybe we should have a common zigbee class where this should go, as the climate sensor probably have this too.